src/vs/workbench/contrib/files/browser/fileActions.contribution.ts

```diff
@@ -200,6 +200,7 @@ const copyRelativePathCommand = {
 appendEditorTitleContextMenuItem(COPY_PATH_COMMAND_ID, copyPathCommand.title, ResourceContextKey.IsFileSystemResource, '1_cutcopypaste');
 appendEditorTitleContextMenuItem(COPY_RELATIVE_PATH_COMMAND_ID, copyRelativePathCommand.title, ResourceContextKey.IsFileSystemResource, '1_cutcopypaste');
 appendEditorTitleContextMenuItem(REVEAL_IN_OS_COMMAND_ID, REVEAL_IN_OS_LABEL, ResourceContextKey.Scheme.isEqualTo(Schemas.file));
+appendEditorTitleContextMenuItem(REVEAL_IN_OS_COMMAND_ID, REVEAL_IN_OS_LABEL, ContextKeyExpr.and(IsWebContext.toNegated(), ResourceContextKey.Scheme.isEqualTo(Schemas.userData)));
```

Comment:
@sandy081 I feel like this command should always be enabled but the other one actually only when running in desktop?

Reply:
Yes, we use `vscode-userdata` scheme everywhere. This will make sure we do not leak in desktop which is not supported else where.

Reply:
@sandy081 ah sorry, yes you are right I was not seeing that the condition is negated.